app/services/idv/profile_maker.rb

```diff
@@ -19,8 +18,8 @@ def save_profile(
       profile.initiating_service_provider = initiating_service_provider
       profile.encrypt_pii(pii_attributes, user_password)
       profile.proofing_components = current_proofing_components
+      profile.verified_at = Time.zone.now unless deactivation_reason # is this necessary to preserve?
```

Comment:
This preserves the current behavior for defining verified_at  ... 
based on the current implementation in prod, it seems :verified_at should be set whether or not the profile is successfully activated?  If this should remain, should it be conditional? 🤔